File: 2026POTD/JANUARY/05-01-2026/1975.cs
public class Solution {
    public long MaxMatrixSum(int[][] matrix) {
        long sum = 0;
        int neg = 0;
        int mn = int.MaxValue;
        foreach (var row in matrix) {
            foreach (var x in row) {
                if (x < 0) neg++;
                sum += Math.Abs((long)x);
                mn = Math.Min(mn, Math.Abs(x));
            }
        }
        if ((neg & 1) == 1) sum -= 2L * mn;
        return sum;
    }
}

File: 2026POTD/JANUARY/05-01-2026/1975..dart
class Solution {
  int maxMatrixSum(List<List<int>> matrix) {
    int neg = 0;
    int mn = 1 << 62;
    int sum = 0;
    for (var row in matrix) {
      for (var x in row) {
        if (x < 0) neg++;
        int ax = x.abs();
        sum += ax;
        if (ax < mn) mn = ax;
      }
    }
    if (neg.isOdd) sum -= 2 * mn;
    return sum;
  }
}